ldk/javascript/readme.md

```diff
@@ -34,8 +34,50 @@ const merged = merge.merge(ldkConfig.default, {
 
 module.exports = merged;
 ```
+#### Development Builds
+We also provide a development Webpack configuration, which offer faster builds in exchange for increased build size:
 
-### Loop Permissions
+```shell
+webpack --entry ./index.js --config ./node_modules/@oliveai/ldk/dist/webpack/config.development.js
+```
+
+```js
+const path = require('path');
+const merge = require('webpack-merge');
+const ldkConfig = require('@oliveai/ldk/dist/webpack/config.development');
+
+const merged = merge.merge(ldkConfig.default, {
+  entry: [path.resolve(__dirname, './index.js')],
+});
+
+module.exports = merged;
+```
+
+
+### Loading a Local Loop Into Olive Helps
+Once you have generated the Loop using the above steps into your `./dist/loop.js` directory, you can now load it into Olive Helps to test.
+
+- Open up Olive Helps and authenticate
```

Comment:
Which wording do you use more frequently with your users? Login or Authenticate?

Reply:
@flowermm  Hmm - good question. I would've opted for login or sign in myself. We haven't established a style guide of our own.